Core/include/Acts/Navigation/MultiLayerNavigationPolicy.hpp

```diff
@@ -48,60 +48,15 @@ class MultiLayerNavigationPolicy : public INavigationPolicy {
                                       const TrackingVolume& volume,
                                       const Logger& logger,
                                       const Config& config,
-                                      indexedUpdatorType grid)
-      : m_volume(volume), m_indexedGrid(std::move(grid)) {
-    ACTS_VERBOSE("Constructing MultiLayerNavigationPolicy for volume "
-                 << m_volume.volumeName());
-
-    // Fill the grid with surfaces
-    std::vector<std::shared_ptr<const Surface>> surfaces = {};
-    for (const auto& surface : m_volume.surfaces()) {
-      if (surface.associatedDetectorElement() == nullptr) {
-        continue;
-      }
-      surfaces.push_back(surface.getSharedPtr());
-    }
-
-    Experimental::detail::CenterReferenceGenerator rGenerator;
-    Experimental::detail::IndexedGridFiller filler{config.binExpansion};
-    filler.fill(gctx, m_indexedGrid, surfaces, rGenerator, {});
-  }
+                                      indexedUpdatorType grid);
```

Comment:
```suggestion
                                      IndexedUpdatorType grid);
```
unrelated to this PR and potentially breaking

Reply:
I am fine with this, but also: using `indexedUpdatorType = IndexedSurfacesNavigation<gridType>;` to
`using IndexedUpdatorType = IndexedSurfacesNavigation<gridType>;`